core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -111,23 +106,17 @@ public SetupWizard(Jenkins j) throws IOException {
     }
 
     /**
+     * Gets the file used to store the initial admin password
+     */
```

Comment:
Why is it public at all? Probably the whole class should just be

```java
@Restricted(NoExternalUse.class)
```

since I cannot foresee any legitimate access from plugins.